doc/classes/Array.xml

```diff
@@ -52,19 +52,19 @@
 		</constructor>
 		<constructor name="Array">
 			<return type="Array" />
-			<param index="0" name="base" type="Array" />
-			<param index="1" name="type" type="int" />
-			<param index="2" name="class_name" type="StringName" />
-			<param index="3" name="script" type="Variant" />
+			<param index="0" name="from" type="Array" />
 			<description>
-				Creates a typed array from the [param base] array.
+				Returns the same array as [param from]. If you need a copy of the array, use [method duplicate].
```

Comment:
Me having to rebase the Array documentation PR for the one billionth time but it's for the best.

Reply:
Current rework removes this change, will see where we land on order, will see :)